.github/workflows/gradle.yml

```diff
@@ -0,0 +1,13 @@
+name: Gradle Tests
+on: [push]
```

Comment:
Can we also not run CI for PRs? Ideally, I'd like to _replace_ AppVeyor with GH actions entirely.

Reply:
I think that once you have it merged, any fork automatically gets these actions, too. So they will automatically run the tests on every commit they make, even before making a PR. I think that's what you're after, right?

Reply:
Kind of. On the one hand, I was confused why GitHub actions weren't present for this PR. I guess we have to merge to `master` first. However, looking at #425, I guess it should [include](https://github.com/jenkinsci/JenkinsPipelineUnit/pull/425/files#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957fR3) `pull_requests` as well.